data/json/snippets.json

```diff
@@ -594,7 +594,431 @@
+        "text" : "\"travelled with a few friends after the thing started.  mostly stuck to bike paths and eaten fruit and mushroomdhfhghghhhh\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"We're the eagle-eyes in the sky!  Just two mutant avians in our helicopter!  Filming the Cataclysm!  Check us out on any working electronics!\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"My friend was getting eaten.  Started ranting about how he had 70 HP on his head, and that the zombie shouldn't penetrate his armor... poor guy.  Bad die roll, I guess.\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"WE'RE GOING TO SAIL TO CANADA, BITCHES!\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"Anyone hear about that guy who tried to sail his family to Canada?  What a moron, right?\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"recently canadian border has gotten more dangerous don't go there\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"Hey, what happened to my dad's airboat?!\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"Reading is good! Never stop reading.  Read EVERYTHING.\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"I'm gonna be honest here, I'm really gonna die soon so I dont want to gt forgnottedn ples dont forntget me y nrmmy name is h@@hbhbh\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"It wasn't Russia or China, guys.  It was me.  It's my fault.  You can just call me... Whales.\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"all it takes to seal a wound is a sawblade and a match!  trust me\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"I wonder what's in this sarcophagus?  Maybe it's a real sarcophagus.  Maybe there's a big guy named God under here.\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"these hulks aint so incredible when ya got .50\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"Got my legs dissected.  It's OK, at least I have my dog!  Her legs were also dissected, but that's OK, we're gonna be OK...\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"Mi-go is friend! Go with Mi-go to friend dimension! On the Friend Team!!\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"We have Landed our Comet. The Sky is Ablaze.\""
+    },{
+        "type" : "snippet",
+        "category" : "note",
+        "text" : "\"My friend died, but at least I made her into a blob turret.\""
```

Comment:
Definitely a Vehicle Additions Pack reference.

Maybe it would be interesting if some mods had relevant snippets for survivor notes. I know that Arcana and Magic Items mod has a snippets section for some other type of note, don't think I've seen more than one show up during play. Seem to only be in the added locations.